pipelines/nextflow/modules/download/download_asm_data.nf

```diff
@@ -36,6 +36,6 @@ process DOWNLOAD_ASM_DATA {
 
     shell:
         '''
-        assembly_download --accession !{meta.accession} --asm_download_dir ./
+        assembly_download --accession !{meta.accession} --download_dir ./ --verbose
```

Comment:
I have a feeling we chatted about this but cannot remember the conclusion: do we want to run always our code with `--verbose` by default? If so it "better" to convert this flag to `--quiet` and set `logging.INFO` as the default level, rather than `logging.WARNING`.